packages/rules-unit-testing/package.json

```
@@ -19,25 +19,12 @@
     "test:ci": "node ../../scripts/run_tests_in_ci.js -s test"
   },
   "license": "Apache-2.0",
-  "dependencies": {
-    "firebase": "8.10.0",
-    "@firebase/component": "0.5.6",
-    "@firebase/logger": "0.2.6",
-    "@firebase/util": "1.3.0",
-    "request": "2.88.2"
-  },
   "devDependencies": {
-    "@google-cloud/firestore": "4.12.3",
-    "@types/request": "2.48.5",
-    "firebase-admin": "9.9.0",
-    "firebase-tools": "9.14.0",
-    "firebase-functions": "3.14.1",
     "rollup": "2.52.2",
     "rollup-plugin-typescript2": "0.30.0"
   },
   "peerDependencies": {
-    "@google-cloud/firestore": "^4.2.0",
-    "firebase-admin": "^9.7.0"
+    "firebase-exp": "^9.0.0-beta.8"
```

```suggestion
    "firebase": "^8.10.0"
```

changeset will update it to v9 automatically when publishing